pr_agent/tools/pr_line_questions.py

```diff
@@ -35,13 +35,17 @@ def __init__(self, pr_url: str, args=None, ai_handler: partial[BaseAiHandler,] =
             "question": self.question_str,
             "full_hunk": "",
             "selected_lines": "",
+            "conversation_history": "",  
         }
         self.token_handler = TokenHandler(self.git_provider.pr,
                                           self.vars,
                                           get_settings().pr_line_questions_prompt.system,
                                           get_settings().pr_line_questions_prompt.user)
         self.patches_diff = None
         self.prediction = None
+        
+        # get settings for use conversation history
+        self.use_conversation_history = get_settings().pr_questions.use_conversation_history
```

Comment:
Since it's only used once, perhaps we could use it directly to reduce code complexity?